test/Sentry.Unity.Tests/UnityApplicationLoggingIntegrationTests.cs

```diff
@@ -56,5 +57,42 @@ public void OnLogMessageReceived_Breadcrumbs_Added()
 
             Assert.AreEqual(2, hub.ConfigureScopeCalls.Count);
         }
+
+        [TestCaseSource(nameof(LogTypesAndSentryLevels))]
+        public void OnLogMessageReceived_UnityErrorLogTypes_CapturedAndCorrespondToSentryLevel(LogType unityLogType, SentryLevel sentryLevel)
+        {
+            var sut = _fixture.GetSut();
+            var hub = _fixture.Hub;
+            sut.Register(hub, SentryOptions);
```

Comment:
Couldn't this:
```
            var hub = _fixture.Hub;
            sut.Register(hub, SentryOptions);
```
be part of the fixture? In `GetSut`?

```suggestion
            var sut = _fixture.GetSut();
```

Reply:
Are you looking at something like this

```csharp
private class Fixture
{
    public TestHub Hub  { get; }

    public UnityApplicationLoggingIntegration Sut { get; }

    public Fixture()
    {
        Hub = new TestHub();
        Sut = new UnityApplicationLoggingIntegration(new TestApplication());
    }
}

[Test]
void Test()
{
  var fixture = new Fixture();
  fixture.Sut.Register(fixture.Hub, SentryOptions);

  fixture.Sut.OnLogMessageReceived("condition", "stacktrace", unityLogType);

  Assert.AreEqual(0, fixture.Hub.CapturedEvents.Count);
}
```